website/static/js/contribManager.js

```diff
@@ -113,6 +116,37 @@ var ContributorModel = function(contributor, currentUserCanEdit, pageOwner, isRe
             id:self.id});
     };
 
+    self.addParentAdmin = function () {
+        // Immediately adds parent admin to the component with permissions=read and visible=True
+        var self = this;
+        $osf.block();
+        self.nodeId = window.contextVars.node.id;
+        self.nodeApiUrl = '/api/v1/project/' + self.nodeId + '/';
+        var url = self.nodeApiUrl + 'contributors/';
+        var userData = JSON.parse(ko.toJSON(self));
```

Comment:
It looks like [`self.serialize()`](https://github.com/CenterForOpenScience/osf.io/pull/7831/files#diff-f4df2c5c528f874302fff1ee05a65e8aR105) is a helper for `ko.toJSON(self)`and could be used instead.

Reply:
Also, I'm not sure that I'm following all of the logic correctly, but is it necessary to serialize the entire view model or would something like `ko.toJSON(self.contributor)` work?

Reply:
There's no self.contributor, but the 'contributor' here I think is a ContributorModel, which is the same thing as self. So probably just `self.serialize()`, like you suggested would work.